src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/util/ExceptionRecordAttachDocumentConnectives.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.util;
+
+import java.util.Set;
+
+public class ExceptionRecordAttachDocumentConnectives {
```

Comment:
Disagree with having such thin wrapper around Set, it is enough to have Set object with a relevant name - e.g. documentConnectivesDuplicates

Reply:
it will contain other value in the following PR - missing documents. the logical connection between duplicates and missing will determine whether there are duplicates or everything got attached and we face the idempotency as expected.

this is doing small refactoring approach so that further implementation doesn't make reviewers cross-eyed

p.s. it is really very difficult to come up with decent name. can always be renamed in the future once bright idea comes in. should not be a blocker here tbh